modin/engines/base/frame/data.py

```diff
@@ -1783,8 +1783,6 @@ def get_axis_lengths(partitions, axis):
             return self._partitions, [o._partitions for o in other], joined_index
 
         base_frame_idx = non_empty_frames_idx[0]
```

Comment:
Why do we define this variable if we only use it in line 1786? Why not use it in line 1789 as well?

Reply:
The code quality in this helper method is in need of some TLC, I do not disagree. Let's put that later.